src/storage/db_interface_statistic.py

```diff
@@ -46,3 +48,6 @@ class StatisticDbViewer(StatisticDb):
 
     def get_statistic(self, identifier):
         return self.statistic.find_one({'_id': identifier})
+
+    def get_statistics(self, *identifiers: str) -> List[dict]:
+        return list(self.statistic.find({'_id': {'$in': identifiers}}))
```

Comment:
The Levenshtein Distance between these two names is 1, making them visually hard to differentiate (especially when mindlessly using autocompletion). Uncle Bob would say: _make meaningful distinctions_ :-).

In my opinion, from both a functional and computational stance, the plural version **could** fully replace the singular version:

1. `get_statistics` basically covers the functionality of `get_statistic`
2. `find_one` is syntactic sugar for `find().limit(1)`
3. the `$in`-operator should have negligible performance impact when `len(identifiers) == 1`

**BUT** we either need to return a `Union[str, List[str]]`, or touch a modest amount of other files in the FACT sources (a quick symbol search yields 11 files).
I guess both proposals exceed the scope of this PR, which is why I wanted to comment, but not request any changes :-)

Reply:
I could rename it for the time being. How about `get_stats_list` or `get_multiple_statistics`?

Reply:
That's better, very much :-)